girder_annotation/test_annotation/web_client_specs/annotationListSpec.js

```diff
@@ -384,8 +385,9 @@ describe('AnnotationListWidget', function () {
             girderTest.waitForLoad();
         });
         it('test header delete link', function () {
-            var $el = $('.g-annotation-list-header');
-            expect($el.find('.g-annotation-delete').length).toBe(1);
+            var $el = $('.g-annotation-list');
+            expect($el.find('th .g-annotation-delete').length).toBe(1);
+            $el.find('th.g-annotation-select input').click();
             $el.find('.g-annotation-delete').click();
```

Comment:
This needs to be changed to `$el.find('th .g-annotation-delete').click();` (and we should do the same for the permissions check above).  Otherwise, we are trying to click ALL of the delete controls at the same time.  Whichever happens to respond first will be the dialog that appears, and if isn't the header control, then we only delete one annotation, not all of them.